src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyCharge = 40 * days;
+  const threeDays = 3;
+  const sevenDays = 7;
```

Comment:
```suggestion
  const longTerm = 7;
```
А якщо період для великої знижки зміниться?